### PR TITLE
these are some REAL tentative numbers trust me - ammo material rebalance (mainly for the ammobench)

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
reduces ammo costs across the board so that the ammo bench isn't so greedy in regards to material when printing ammo. still, please don't lock in turbo unless necessary.

ammo costs:
- reduced from base `SMALL_MATERIAL_COST * 5` to `SMALL_MATERIAL_COST * 2`
(this is because the old small cost was 5% of a sheet. the new small cost is 20% of a sheet. with the old multiplier a single casing was literally 50% a sheet (bad) and now it's just 20% a sheet (maybe slightly less bad))
- adjusts all material costs of ammo to be in that same ballpark (all totals up to 2 total `SMALL_MATERIAL_COST`)
- shotgun ammo base material total nudged to `SMALL_MATERIAL_COST * 4` (mild decrease from literally half a sheet per shell. i don't know if i want to decrease this considering how shotguns are but idk)

## How This Contributes To The Skyrat Roleplay Experience
does shooting people still count as roleplay? makes a single bullet no longer cost HALF AN ENTIRE METAL SHEET

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

off a t4 bench:
image
off a t1 bench:
image

</details>

## Changelog

:cl:
balance: Base material costs for ammunition have been sharply reduced across the board, because a single 9mm casing or equivalent costing half a sheet was egregious. This mainly only applies to the ammo bench; lathe recipes are their own system and remain untouched.
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
Ammo has always has a very high material cost, like many printable combat items (see: autolathe costs)

I'm not against a high material cost for ammo, especially for the ammo bench which can just have the parts upgraded to lower it.

**Comment by HoodMilk** [comment]
With the TG mineral rebalance ammo is actually pretty cheap from lathes, I think a 357 speedloader was only 360 cm3 iron, and that's including the speedloader.

More consistency with ammo costs is always good.

**Comment by YakumoChen** [comment]
> With the TG mineral rebalance ammo is actually pretty cheap from lathes, I think a 357 speedloader was only 360 cm3 iron, and that's including the speedloader.
> 
> More consistency with ammo costs is always good.

https://github.com/Skyrat-SS13/Skyrat-tg/blob/ca066b1b2832477a839e86c0f4e5dbb3dba70471/code/modules/research/designs/autolathe/security_designs.dm#L181

I'm pretty sure that's false? It's 2 sheets of iron per casing, no speedloader.

The real consistency would be making bullets expensive.

File: modular_skyrat/modules/shotgunrebalance/code/shotgun.dm [hunk 1]
```diff
@@ -1,7 +1,97 @@
...
+/obj/item/ammo_casing/shotgun/ion
+	can_be_printed = FALSE // techshell. assumed intended balance being a pain to assemble
+
+/obj/item/ammo_casing/shotgun/incapacitate
+	name = "hornet's nest shell"
+	desc = "A 12 gauge shell filled with some kind of material that excels at incapacitating targets. Contains a lot of pellets, \
+	sacrificing individual pellet strength for sheer stopping power in what's best described as \"spitting distance\".\
+	<br><br>\
+	<i>HORNET'S NEST: Fire an overwhelming amount of projectiles in a single shot.</i>"
+	// ...you know what if you're confident you can get up in there, you might as well get to use it if you're able to print Weird Shells.
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/shotgunrebalance/code/shotgun.dm:94]:
```suggestion
	// ...you know what if you're confident you can get up in there, you might as well get to use it if you're able to print Weird Shells.

```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Just a minor thing, otherwise it seems good to me. I'd probably just test-merge it first, just in case anything was to suddenly go wrong, but I doubt it'll be the case.

**Review by GoldenAlpharex — APPROVED** [review]
Should be good for a test-merge.

**Comment by GoldenAlpharex** [comment]
Haven't heard any complaints, or really much feedback at all, I doubt anyone's actually got an issue with this. Let's go ahead and merge it now then.

**Pull request merged**